.github/workflows/ci.yml

```diff
@@ -13,10 +13,11 @@ jobs:
     strategy:
       matrix:
         flavour: [
-          assembleOculusvrArm64WorldGecko,
-          assembleHvrArm64WorldGecko,
-          assemblePicoxrArm64WorldGecko,
-          assembleLynxArm64WorldGecko
+          assembleOculusvrArm64WorldGenericGecko,
+          assembleOculusvrArm64WorldMetaStoreGecko,
+          assembleHvrArm64WorldGenericGecko,
+          assemblePicoxrArm64WorldGenericGecko,
+          assembleLynxArm64WorldGenericGecko
```

Comment:
Ordering should be swapped. GenericGecko -> GeckoGeneric.

Reply:
oh very true :)